testing/unittests/dalitests.cpp

```diff
@@ -3265,7 +3265,150 @@ class DaliSysInfoLoggerTester : public CppUnit::TestFixture
 CPPUNIT_TEST_SUITE_REGISTRATION( DaliSysInfoLoggerTester );
 CPPUNIT_TEST_SUITE_NAMED_REGISTRATION( DaliSysInfoLoggerTester, "DaliSysInfoLoggerTester" );
 
+class FileReadPropertiesUpdaterTester : public CppUnit::TestFixture
+{
+    CPPUNIT_TEST_SUITE(FileReadPropertiesUpdaterTester);
+        CPPUNIT_TEST(testInit);
+        CPPUNIT_TEST(testFileReadPropertiesUpdater);
+    CPPUNIT_TEST_SUITE_END();
+
+    #define SCOPE "testfileupdate"
+    std::vector<std::string> subFileNames = { "subfile1", "subfile2", "subfile3" };
+    std::vector<std::string> superFileNames = { "superfile", "supersuperfile" };
+    std::string getSuperFileLogicalName(unsigned superfileNum)
+    {
+        return std::string(SCOPE) + "::" + superFileNames[superfileNum - 1];
+    }
+    std::string getSubFileLogicalName(unsigned subfileNum)
+    {
+        return std::string(SCOPE) + "::" + subFileNames[subfileNum - 1];
+    }
```

Comment:
instead of using access function to reconstruct the required compound filename, could just have them in the vector and access them directly (nothing uses the non-compound versions).
e.g. like this:
```
    static constexpr const char *scope = "testfileupdate";
    std::vector<std::string> subFileNames = {
        std::string(scope) + "::subfile1",
        std::string(scope) + "::subfile2",
        std::string(scope) + "::subfile3"
    };
    std::vector<std::string> superFileNames = {
        std::string(scope) + "::superfile",
        std::string(scope) + "::supersuperfile"
    };
```